src/east/system_commands/sys_setup.py

```diff
@@ -82,6 +83,43 @@ def _get_toolchain_download_link():
     return link
 
 
+def _get_cppcheck_download_link():
+    """Just a convenience function for getting the link for the Cppcheck source code.
+    This link will be updated regularly.
+
+    Note: If you change version do not forget to update the folder name in the
+    _install_cppcheck below!
+    """
+
+    link = "https://github.com/danmar/cppcheck/archive/refs/tags/2.12.0.tar.gz"
```

Comment:
You could make the link, or at least the version a constant in the `constans.py` file. Then, this function and `_install_cppcheck` would use the same "source of truth".
You could then remove the Note and will also never forget to update `_install_cppcheck`

Reply:
I removed the explicit cppcheck version from the `__install_cppcheck` function and replaced it with glob function.

The purpose of constants.py is a bit different, I only store there various paths that should not ever change, but should be available to the rest of the East.

The version links are used only in sys_setup.py file.